docs/docs/Components/components-tools.md

```diff
@@ -628,13 +628,13 @@ Deprecated components have been replaced by newer alternatives and should not be
 ### MCP Tools (stdio)
 :::important
 This component is deprecated as of Langflow version 1.3.
-Instead, use the [MCP server component](/components-tools#mcp-server)
+Instead, use the [MCP server component](/components-tools#mcp-connection)
```

Comment:
Shouldn't the text in the brackets say "MCP connection component"?

Reply:
I forgot the deprecated ones, good catch